### PR TITLE
deployment of customizable log4j.properties …

### PR DESCRIPTION
**Pull request by lhoss**
…(logger levels defaults preset from the official spark log4j template)

ps: I'm thinking about adding an extra toggle variable for this feature like `spark_log4j_deploy_enabled`

### CODE REVIEW

**Comment by lhoss** [comment]
@tnation14 @hectcastro ?

File: defaults/main.yml [hunk 1]
```diff
@@ -21,3 +21,25 @@ spark_hive_site_properties: {}
 
 spark_env_extras: {}
 spark_defaults_extras: {}
+
+# Logger levels set to the official defaults
+# Ref: https://github.com/apache/spark/blob/master/conf/log4j.properties.template
+spark_log4j_loggers:
```
Comment (lhoss) [inline defaults/main.yml:27]: we could also use a dict instead (the minimally verbose variant):
```
 log4j.logger.org.apache.spark.repl.Main: WARN
 log4j.logger.org.spark_project.jetty: WARN
...
```
or a list of lines
```
- "log4j.logger.org.apache.spark.repl.Main: WARN"
- "log4j.logger.org.spark_project.jetty: WARN"
```

Comment (tnation14) [reply defaults/main.yml:27]: I think this is actually OK as is.

**Review by tnation14 — APPROVED** [review]
LGTM 👍

I don't think it's unnecessary to add each dictionary key in `spark_log4j_loggers` to the README, but I think it would be useful to explain what the variable does at a high level.

**Comment by lhoss** [comment]
sure, will document a 1 liner in the README now ( tough that might need another iteration for the other PRs where I'll also update the README !):
```
- `spark_log4j_loggers` - A list of dictionaries configuring the spark log4j loggers (default: logger config from spark/conf/log4j.properties.template)
```

**Pull request merged**